cogeo_mosaic/mosaic.py

```diff
@@ -97,13 +99,15 @@ def _create_mosaic(
         quadkey_zoom: int, optional
             Force mosaic quadkey zoom.
         accessor: callable, required
-            Function called on each feature to get its identifier (default is feature["properties"]["path"])
+            Function called on each feature to get its identifier (default is feature["properties"]["path"]).
+        asset_filter: callable, required
```

Comment:
This is definitely a really advanced feature because any `custom filter method` should match the inputs of the actual method (e.g `tile, intersect_dataset, intersect_geoms`) I'm fine with this but we will need documentation

Reply:
Yes, docs required